index.html

```diff
@@ -0,0 +1,206 @@
+<html>
+    <head>
+        <title>YouTube</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body> 
+
+<nav class="flex-div">
+    <div class="nav-left flex-div">    
+    <img src="images/menu.png" class="menu-icon">
+    <img src="images/logo.png" class="logo">
+        </div>
+        <div class="nav-middle flex-div">
+            <div class="search-box flex-div">
+
+                <input type="search" placeholder="search">
+                <img src="images/search.png" class="search-icon">
+            </div>
+            <img src="images/voice-search.png" class="mic-icon">
+        </div>
+        <div class="nav-right flex-div">
+            <img src="images/upload.png">
+            <img src="images/more.png">
+            <img src="images/notification.png">
+            <img src="images/Jack.png" class="user-icon">
+            
+        </div>
+    </nav>
+
+
+    <!-- -----------side-bar------------- -->
+
+    <div class="side-bar">
+        <div class="shortcut-links">
+            <a href="#"><img src="images/home.png"><p>Home</p></a>
+            <a href="#"><img src="images/explore.png"><p>Explore</p></a>
+            <a href="#"><img src="images/subscriprion.png"><p>Subscription</p></a>
+            <a href="#"><img src="images/library.png"><p>Library</p></a>
+            <a href="#"><img src="images/history.png"><p>History</p></a>
+            <a href="#"><img src="images/playlist.png"><p>Playlist</p></a>
+            <a href="#"><img src="images/messages.png"><p>Messages</p></a>
+            <a href="#"><img src="images/show-more.png"><p>Show More</p></a>
+            <hr>
+        </div>
+            <div class="subscribed-list">
+                <h3>SUBSCRIBED</h3>
+                <a href="#"><img src="images/Jack.png"><p>Jack Man</p></a>
+                <a href="#"><img src="images/megan.png"><p>Megan Foxy</p></a>
+                <a href="#"><img src="images/cameron.png"><p>Cameron kalley</p></a>
+                <a href="#"><img src="images/tom.png"><p>Tom Croissant</p></a>
+                <a href="#"><img src="images/simon.png"><p>Simon Showmen</p></a>
+            </div>
+        
+    </div>
+<!-- -------- Main--------- -->
+<div class="container">
+ <div class="banner">
+      <img src="images/banner.png">
+ </div>
+<div class="list-container">
+    
+    <div class="vid-list">
+        <a href="#">
+            <iframe class="thumbnail" width="374" height="212"
+             src="https://www.youtube.com/embed/O_qgaOUR3Fo" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+             allowfullscreen></iframe>
+        </a>
+    <div class="flex-div">
+        <img src="images/Jack.png">
+        <div class="vid-info">
+            <a href="#">Best Practices of Ecommerce Bussiness</a>
+            <p>Jack Man</p>
+            <p>15k views &bull; 2 days</p>
+        </div>
```

Comment:
- [ ] [OPTIONAL] I would suggest using the `<li>` tag instead of the `<div>` tag.
 `<li>` means an item is a list that lets parsers (browsers, search engines, spiders) know that you're listing items. You can use DIV instead of LI but those parsers would never know that those items are being listed and DIV does not really describe anything except that it's a block.